features/api-token.feature

```diff
@@ -0,0 +1,43 @@
+@apitoken
+Feature: User API Token
+
+    A user API token can be created and used in place of a password when
+    interacting with the API. This would allow users to obtain valid JWTs and
+    interact with the Screwdriver API in a programmatic way.
+
+    Rules:
+        - Tokens should be specifically tied to the user that issued them
+        - Tokens should have a label/name associated with them
+        - Users can add a description to their issued tokens, so they know what they're using them for
+        - User can see when a token was last used
+        - Users can revoke a token at any time
+        - No one can ever look up the raw value after initially generating the token
+
+    Background:
+        Given the user "calvin" exists
+
+    @ignore
+    Scenario: New API Token
```

Comment:
`Add` or `Generate New API Token`? Just to be consistent with other scenarios~ They all start with verb